app.py

```diff
@@ -1,17 +1,62 @@
-from flask import Flask, render_template
+import os
+import urllib.request
+import json
+import sys
+import getopt
+import shutil
+import glob
+from flask import Flask, flash, request, redirect, render_template, url_for, send_from_directory
+from werkzeug.utils import secure_filename
 
-UPLOAD_FOLDER = './uploads'
+# ALLOWED_EXTENSIONS = set(['json'])
 
 app = Flask(__name__)
-app.secret_key = "secret key"
-app.config['UPLOAD_FOLDER'] = UPLOAD_FOLDER
-app.config['MAX_CONTENT_LENGTH'] = 16 * 1024 * 1024
+app.config.from_object("settings.config.DevelopmentConfig")
 
+def allowed_file(filename):
+    name, extension = os.path.splitext(filename)
+    extension = extension[1:].lower()
+    return extension in app.config['ALLOWED_EXTENSIONS']
 
 @app.route('/')
 def index():
     return render_template('home.html')
 
+@app.route('/convert')
+def upload_form():
+    return render_template('convert.html')
+
+@app.route('/', methods=['POST'])
+def upload_file():
+    convert_from = request.form["convertFrom"]
+    convert_to = request.form["convertTo"]
+    upload_files = glob.glob('./uploads/*')
+    for f in upload_files:
+        os.remove(f)
+    # check if the post request has the file part
+    if 'file' not in request.files:
+        flash('No file part')
+        return redirect(request.url)
+    file = request.files['file']
+    if file.filename == '':
+        flash('No file selected for uploading')
+        return redirect(request.url)
+    if file and allowed_file(file.filename):
+        filename = secure_filename(file.filename)
+        file.save(os.path.join(app.config['UPLOAD_FOLDER'], filename))
+        os.system('python ./' + convert_from + '2' + convert_to + '.py')
```

Comment:
This is ok for now, but the next PR should address this. We shouldn't be calling the OS to call python to run a script. Make converter a decoupled, package-able component that gets called in code via a simple function call.

Reply:
Sounds good